modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/events/ClientExtensionsServicePreAction.java

```diff
@@ -80,20 +85,10 @@ public void run(
 			return;
 		}
 
-		themeDisplay.setFaviconURL(_getFaviconURL(layout));
-
-		ThemeCSSCET themeCSSCET = _getThemeCSSCET(layout);
+		_setThemeDisplayCSSURLs(
```

Comment:
Is there a reason this is now coming before `themeDisplay.setFaviconURL(_getFaviconURL(layout));`

Reply:
No, my mistake